src/mustache_core.js

```diff
@@ -381,7 +381,7 @@ var core = {
 				else if(state.text && !valueShouldBeInsertedAsHTML(value)) {
 					//!steal-remove-start
 					if (process.env.NODE_ENV !== 'production') {
-						if(value !== null && typeof value === "object") {
+						if(value !== null && typeof value === "object" && canReflect.isBuiltIn(value) === false) {
```

Comment:
Can you just use `!canReflect.isBuiltIn(value)`?

Reply:
Done!